plonecli/cli.py

```diff
@@ -61,7 +61,9 @@ def cli(context, list_templates, versions):
     @click.argument('name')
     @click.pass_context
     def create(context, template, name):
-        """Create a new Plone package"""
+        """Create a new Plone package.
+        Run "Plonecli -l" to see available templates
```

Comment:
Use `\n` for new line.

Reply:
@kakshay21 I have shifted that message to a new line.